.pipelines/singletenancy/dualstack-overlay/dualstackoverlay-e2e-step-template.yaml

```diff
@@ -115,3 +117,52 @@ steps:
     name: "restartCNS_ValidatePodState"
     displayName: "Restart CNS and Validate Pod State"
     retryCountOnTaskFailure: 3
+
+  - ${{ if eq( parameters['testDropgz'], true) }}:
+      - script: |
+          echo "##vso[task.setvariable variable=dropgzVersion]$(make cni-dropgz-test-version)"
+        name: dropgzTestVersion
+        displayName: "Dropgz Test Version"
+  - ${{ else }}:
+      - script: |
+          echo "##vso[task.setvariable variable=dropgzVersion]$(make cni-dropgz-version)"
+        name: dropgzVersion
+        displayName: "Dropgz Version"
```

Comment:
These should be moved above line 39 and also be changed in task `integrationTest` to use the dropgz veriosn variable.  Currently the pipeline can test different dropgz/cni in these tasks based on the parameter(test dropgz) in a single run.